packages/flame/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,179 @@
+import 'package:flutter/gestures.dart';
+
+import '../../extensions.dart';
+
+abstract class BaseInfo<T> {
+  final T raw;
+
+  BaseInfo(this.raw);
+}
+
+abstract class PositionInfo<T> extends BaseInfo<T> {
```

Comment:
how about SinglePositionInfo ? because there might be events with multiple positions that would not use this

Reply:
Added